pkg/handlers/config/config.go

```diff
@@ -1,69 +1,213 @@
 package config
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
 
+	auth0 "github.com/aserto-dev/aserto/pkg/auth0/api"
 	"github.com/aserto-dev/aserto/pkg/cc"
+	"github.com/aserto-dev/aserto/pkg/cc/config"
 	"github.com/aserto-dev/aserto/pkg/handlers/user"
 	"github.com/aserto-dev/aserto/pkg/jsonx"
 	"github.com/aserto-dev/aserto/pkg/keyring"
-	api "github.com/aserto-dev/go-grpc/aserto/api/v1"
-	account "github.com/aserto-dev/go-grpc/aserto/tenant/account/v1"
+	"github.com/aserto-dev/aserto/pkg/x"
+	"github.com/aserto-dev/clui"
+	"github.com/aserto-dev/go-grpc/aserto/api/v1"
+	"github.com/aserto-dev/go-grpc/aserto/tenant/account/v1"
 
 	"github.com/pkg/errors"
 )
 
-type GetEnvCmd struct{}
+type GetContextsCmd struct{}
 
-func (cmd *GetEnvCmd) Run(c *cc.CommonCtx) error {
-	return jsonx.OutputJSON(c.UI.Output(), c.Environment)
+func (cmd *GetContextsCmd) Run(c *cc.CommonCtx) error {
+	configFile, err := config.GetConfigFile()
```

Comment:
If we're running a command, doesn't that mean we already loaded the config file? Why load it again here?